tests/test_videos.py

```diff
@@ -205,13 +164,13 @@ def test_compression_works(self, high_res_video):
         assert video_file.get_preset() == format_presets.VIDEO_LOW_RES, 'Should have low res preset'
 
 
-    def test_compression_works(self, high_res_video):
+    def test_compression_max_width_works(self, high_res_video):
         video_file = make_video_file(high_res_video)
-        video_file.ffmpeg_settings = {'crf': 33, 'max_height': 300}
+        video_file.ffmpeg_settings = {'crf': 33, 'max_width': 200}
         video_filename = video_file.process_file()
         video_path = config.get_storage_path(video_filename)
         width, height = get_resolution(video_path)
-        assert height == 300, 'should be compress to 300 v resolution'
+        assert width == 200, 'should be compress to 200 hz resolution'
         assert video_file.get_preset() == format_presets.VIDEO_LOW_RES, 'Should have low res preset'
```

Comment:
This is a completely unrelated change:  `pyflakes` pointed out that the function `test_compression_works` was duplicate (probably copy-paste when I wrote these initially). 

Instead of deleting duplicate, I made the test check the `max_width` option.